packages/manager/core/generator/app/templates/src/components/GuideLink/guide-utils.tsx

```diff
@@ -0,0 +1,125 @@
+import { useState, useEffect } from 'react';
+import { Region, CountryCode } from '@ovh-ux/manager-config';
+import {
+  useAuthentication,
+  useEnvironment,
+} from '@ovh-ux/manager-react-core-application';
+
+const docUrl = 'https://docs.ovh.com/';
+
+type GuideLinks = { [key in CountryCode]: string };
+type BaseUrlProps = { [key in Region]: Partial<GuideLinks> };
+
+const baseUrlPrefix: BaseUrlProps = {
+  EU: {
+    DE: `${docUrl}de`,
+    ES: `${docUrl}es`,
+    IE: `${docUrl}ie/en`,
+    IT: `${docUrl}it`,
+    PL: `${docUrl}pl`,
+    PT: `${docUrl}pt`,
+    FR: `${docUrl}fr`,
+    GB: `${docUrl}gb/en`,
+  },
+  CA: {
+    CA: `${docUrl}ca/en`,
+    QC: `${docUrl}ca/fr`,
+    WE: `${docUrl}us/en`,
+    WS: `${docUrl}us/en`,
+  },
+  US: {
+    US: `${docUrl}us/en`,
+  },
+};
```

Comment:
**nitpick:** Logic to build URL can be part of any reusable component.

Reply:
I create a reusable component but the dev can custom the baseUrl and create a new variable 

exemple :  
const apiUrl =  = 'https://api.docs.ovh.com/';

ES: `${apiUrl}es`,